API.md

```diff
@@ -18,7 +18,59 @@ _Returns:_ A string representing a bearer token that can be used to carry out op
 
 ### Scheme: OpenID Connect (OIDC)
 
-Coming soon ...
+For authenticating with OpenID Connect, there are two methods:
+
+- Resource Owner Password Credentials Grant
+- Authorization Code Grant
+
+#### Resource Owner Password Credentials Grant
+
+The core advantage of this grant is the flexibility in how it may be applied.
+
+Following is a brief example of authenticating from the command line.
+
+##### Request
+```bash
+username_and_password=$(echo -n "<username>:<password>" | base64 -)
+curl -X POST -H "Authorization: iRODS $username_and_password" \
+    http://localhost:<port>/irods-http-api/<version>/authenticate
+```
+
+##### Response
+A string representing a bearer token that can be used to carry out operations as the authenticated user.
```

Comment:
`used to carry out operations` -> `used to execute operations`

Reply:
Done.